app/views/shared/_output.html.erb

```diff
@@ -8,13 +8,13 @@
       </div>
 
       <button class="btn btn-default only-finished" id="output-grow-toggle">Expand</button>
-      <%= link_to "Download log", project_deploy_path(@project, @deploy, format: :text), class: "btn btn-primary only-finished" %>
+      <%= link_to "Download log", project_deploy_path(@project, deployable, format: :text), class: "btn btn-primary only-finished" %>
```

Comment:
nvm, still using the ivar in https://github.com/zendesk/samson/pull/784/files#diff-00955b7a831930544081954ec53bf6bfL4 (was thinking we may be able to make it a local if it was assigned in that method)